platform/x11/context_gl_x11.cpp

```diff
@@ -179,9 +179,18 @@ int ContextGL_X11::get_window_height() {
 }
 
 void ContextGL_X11::set_use_vsync(bool p_use) {
-	GLXDrawable drawable = glXGetCurrentDrawable();
-	//GLXSwapIntervalEXT(x11_display, drawable, p_use?1:0);
-	use_vsync=p_use;
+	static bool setup = false;
+	static PFNGLXSWAPINTERVALEXTPROC glXSwapIntervalEXT = NULL;
+
+	if (!setup) {
+		setup = true;
+		glXSwapIntervalEXT = (PFNGLXSWAPINTERVALEXTPROC) glXGetProcAddressARB((const GLubyte*)"glXSwapIntervalEXT");
+	}
+	if (glXSwapIntervalEXT) {
```

Comment:
IMO it's unnecessary as `use_vsync` can't be true when we don't have the method.